server/stream.go

```diff
@@ -297,14 +302,25 @@ func (mset *Stream) setLeader(isLeader bool) error {
 }
 
 // Lock should be held.
-func (mset *Stream) startSyncSub() {
+func (mset *Stream) startClusterSubs() {
+	if mset.infoSub == nil {
+		if jsa := mset.jsa; jsa != nil {
+			isubj := fmt.Sprintf("$JSC.SI.%s.%s", jsa.acc(), mset.config.Name)
```

Comment:
If we make this a constant, replace the string here.

Reply:
Done